nms/babel.config.js

```diff
@@ -12,7 +12,7 @@
  */
 
 function isWebpack(caller) {
-  return !!(caller && caller.name === 'web');
```

Comment:
`caller.target` was `web` so the plugin was never enabled. Somehow it work without, but I kept it because the docs say you should use it.